test/metabase/test/data/interface.clj

```diff
@@ -793,3 +793,13 @@
   ;; Following cyclic dependency by that requiring resolve.
   ((requiring-resolve 'metabase.test.data.impl/resolve-dataset-definition)
    'metabase.test.data.dataset-definitions 'test-data))
+
+(defmulti native-query-with-card-template-tag
```

Comment:
Added this method so third-party driver authors can theoretically write different implementations, even tho AFAIK in practice only SQL drivers support `:native-parameter-card-reference`